src/libcore/sync/atomic.rs

```diff
@@ -1550,12 +1550,28 @@ unsafe fn atomic_xor<T>(dst: *mut T, val: T, order: Ordering) -> T {
 
 /// An atomic fence.
 ///
+/// A fence creates happens-before edges between other atomic operations or
+/// fences when the certain condition is met. Also, a fence prevents reordering
+/// of operations around a fence by CPU or a compiler. Each behavior is
+/// controlled by the value of [`Ordering`].
+///
 /// A fence 'A' which has [`Release`] ordering semantics, synchronizes with a
 /// fence 'B' with (at least) [`Acquire`] semantics, if and only if there exists
 /// atomic operations X and Y, both operating on some atomic object 'M' such
 /// that A is sequenced before X, Y is synchronized before B and Y observes
 /// the change to M. This provides a happens-before dependence between A and B.
 ///
+///     Thread 1                                          Thread 2
```

Comment:
Travis build is failing due to this line. I removed this line on my local environment and ./x.py test src/libcore completed successfully. Is this a bug, or intended?